ROM.sh

```diff
@@ -1440,7 +1440,7 @@ function the_start() # 0
 
     if [[ "$BRANCH" =~ (master|staging) ]]; then
         # Download the Remote Version of Updater, determine the Internet Connectivity by working of this command
-        curl -fs -o upScriBt.sh https://raw.githubusercontent.com/a7r3/ScriBt/${BRANCH}/upScriBt.sh && \
+        curl -fs -o ${PATHDIR}upScriBt.sh https://raw.githubusercontent.com/a7r3/ScriBt/${BRANCH}/upScriBt.sh && \
```

Comment:
Oh, wait, keep it.

Else move line 1439 (`cd` back to current dir) just after the `if` condition